wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -482,29 +483,33 @@ public void run() {
             stoppedBy);
         publishWorkspaceStatusEvent(workspaceId, STOPPED, STOPPING, null);
       } catch (ServerException | InfrastructureException e) {
-        // remove before firing an event to have consistency between state and the event
```

Comment:
Can you elaborate on why we don't need this comment anymore?

Reply:
Well, I have no idea =)
I guess it was removed by mistake. I'll revert it

Reply:
Reverted